source/fem-dem/postprocessing_cfd_dem.cc

```diff
@@ -0,0 +1,128 @@
+// Base
+#include <deal.II/base/quadrature_lib.h>
+
+// Lac
+#include <deal.II/lac/dynamic_sparsity_pattern.h>
+#include <deal.II/lac/vector.h>
+
+// grid
+#include <deal.II/grid/grid_tools.h>
+
+// Dofs
+#include <deal.II/dofs/dof_accessor.h>
+#include <deal.II/dofs/dof_handler.h>
+
+// Lac - Trilinos includes
+#include <deal.II/lac/trilinos_parallel_block_vector.h>
+#include <deal.II/lac/trilinos_vector.h>
+
+// Fe
+#include <deal.II/fe/fe_q.h>
+#include <deal.II/fe/fe_system.h>
+#include <deal.II/fe/fe_values.h>
+#include <deal.II/fe/mapping_q.h>
+
+// Lethe includes
+#include <core/boundary_conditions.h>
+#include <core/parameters.h>
+#include <core/rheological_model.h>
+#include <core/vector.h>
+
+#include <fem-dem/postprocessing_cfd_dem.h>
+
+
+using namespace dealii;
+
+
+template <int dim, typename VectorType>
+std::pair<double, double>
+calculate_total_volume(const DoFHandler<dim> &void_fraction_dof_handler,
+                           const VectorType   &present_void_fraction_solution,
+                           const Quadrature<dim> &quadrature_formula,
+                           std::shared_ptr<Mapping<dim>>    mapping)
```

Comment:
@blaisb You could parse the argument as Mapping and dereference here, no?. Additionally, would it be possible to make it const?

Reply:
I did not word it correctly but that's what I meant :)